client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -56,6 +59,7 @@ class BusinessDetails extends Component {
 		await updateProfileItems( {
```

Comment:
I think unrelated to this branch, but I'm getting an invalid param for `other_platform` being an empty string.  Maybe we can add an empty string as an option or filter out empty values.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2763/commits/19ae63deb6c634599d4b28df76c19cc67c876543.